docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.7'
```

Comment:
This file is probably not needed anymore, same as the added `Dockerfile`s?

Reply:
Yep we can.